docs/modules/traits/pages/toleration.adoc

```diff
@@ -0,0 +1,44 @@
+= Toleration Trait
+
+// Start of autogenerated code - DO NOT EDIT! (description)
+This trait will set a Toleration over an integration pod. Tolerations allow (but do not require) the pods to schedule onto nodes with matching taints.
+See https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ for more details.
+
+The toleration should be expressed in a similar manner of taints *_Key[=Value]:Effect[:Seconds]_* where values in square brackets are optional. Examples:
+
+my-key:NoExecute
```

Comment:
I'd suggest to add actual examples, like adding toleration to schedule pods on master nodes, e.g. `node-role.kubernetes.io/master:NoSchedule`.